core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -666,6 +666,10 @@ public enum CoreError implements ScalarDbError {
           + "Primary-key columns must not contain any of the following characters in Cosmos DB: ':', '/', '\\', '#', '?'. Value: %s",
       "",
       ""),
+  CONSENSUS_COMMIT_INSERTING_ALREADY_WRITTEN_DATA_NOT_ALLOWED(
+      Category.USER_ERROR, "0146", "Inserting already-written data is not allowed", "", ""),
+  CONSENSUS_COMMIT_DELETING_ALREADY_INSERTED_DATA_NOT_ALLOWED(
+      Category.USER_ERROR, "0147", "Deleting already-inserted data is not allowed", "", ""),
```

Comment:
@josh-wong Sorry, could you please review this message as well? 🙇

Reply:
@brfrn169 I ran a small code to reproduce this error. I noticed the namespace, table and key of the target record weren't shown in any error message. How about adding those information into this error message? I think this is a nice-to-have improvement and won't block this PR from being approved, though.

Reply:
@brfrn169 

> @josh-wong Sorry, could you please review this message as well? 🙇

LGTM!👍